audit/buckets/kubernetes-public.kubernetes_public_billing.txt

```diff
@@ -0,0 +1,156 @@
+gs://kubernetes_public_billing/billing--2019-01-10.csv
```

Comment:
We probably should not be dumping bucket contents or VM names or anything that is likely to be constantly changing.  Unlikely that any 2 audits could produce the same data for this.